web-components/src/merch-card.js

```diff
@@ -110,10 +111,12 @@ export class MerchCard extends LitElement {
 
     updated(changedProperties) {
         if (
-            changedProperties.has('badgeBackgroundColor') &&
+            (changedProperties.has('badgeBackgroundColor') || changedProperties.has('borderColor')) &&
```

Comment:
IMO this desserves a unit test. Logic is getting complicated.

Reply:
unit test added